### ISSUE TITLE
Megamenu links are not keyboard accessible

### ISSUE THREAD

**Issue by biljanap** [wet-boew/GCWeb#149]
This would fail Guideline 2.1 Keyboard Accessible (http://www.w3.org/TR/WCAG20/#keyboard-operation)
Right now the user can only tab to “Jobs” (first link of first level links) because the tabindex is set to 0 (tabindex=”0”). Other links (including sublinks) do not participate in the tabbing flow of the document because their tabindex is given a value of -1 (tabindex=”-1”).
If given a value of "-1", the element can't be tabbed to and it can’t be focused via the keyboard.
